tempus_fugit_minecraft/main.py

```diff
@@ -179,10 +179,9 @@ def _initialize(self):
                 s -= d  # decrement side length so hills taper off
 
         # generate_clouds was here.
-        clouds = self.generate_clouds(n, 150)
-        for cloud in clouds:
-            for x, c, z in cloud:
-                self.add_block((x, c, z), LIGHT_CLOUD, immediate=True)
+        clouds = self.generate_clouds_positions(n, 150)
```

Comment:
I'm assuming 150 is the value for `num_of_clouds`, so why not change it to `generate_clouds_positions(n, num_of_clouds=150)`? This would help the code's readability at a glance instead of looking at the function's definition to see what that parameter does.